postgres/cloudpostgres/cloudpostgres_test.go

```diff
@@ -71,3 +75,62 @@ func TestOpen(t *testing.T) {
 		t.Error("Close:", err)
 	}
 }
+
+func TestOpenBadValue(t *testing.T) {
+	// This test will be skipped unless the project is set up with Terraform.
+
+	tfOut, err := terraform.ReadOutput(".")
+	if err != nil {
+		t.Skipf("Could not obtain harness info: %v", err)
```

Comment:
My intent was to give it otherwise valid values, but only altering the parameters given to the function. I understand that in the current implementation of the function, it would abort earlier in the process, but I'm worried about more the case of the system under test not checking for these conditions, where it would produce an error for unrelated reasons.